src/platform/apollolake/include/platform/memory.h

```diff
@@ -352,6 +352,7 @@
 /* LP SRAM */
 #define LP_SRAM_BASE		0xBE800000
 #define LP_SRAM_SIZE		0x00020000
+#define LP_EBB_MASK		0x03
```

Comment:
We already have LPSRAM_MASK in platcfg.h.

Reply:
Right, let's apply that one.